### DISCUSSION TITLE
PvP enhancements

### DISCUSSION THREAD

**Discussion by dumikachromiecraft** [chromiecraft/chromiecraft#6655] [Suggestions]
Hello,

I have played here for the past 3 years, and I have some suggestions that I believe could greatly enhance PvP QoL and participation. These are the following:

1. Kill the queue time between arena games with fire. Seriously. There is no good reason for this to be in with such low arena activity. I only heard complaints about it.
2. The 25 points from CTA and the 10 points from daily wins are appreciated, and while they increase BG activity, they don't do much about arena activity. I propose that you give 10 AP for the first daily arena win per category (2s/3s/5s). This could bring the total to 380 per week from dailies (70 per BG/100 per CTA/210 from arena). That's 724 points per week, so you could get a piece every 3 weeks not every 6 weeks like now.
3. Release the next tier rings and necklace now, and release the accessories (belt, bracer, boots) with TK. Currently the best PvP rings are from SSC, while actual PvP players are stuck with rings from 5 brackets ago. The next tier jewelry are item level 128, on par with SSC gear.
4. The main problem with climbing the rankings is population. I have many times reached high rating only to abandon my team, because the partner quit the server from low activity. And the grind starts from 0 again in a new team. Make the points start from 1500. If team is bad, they will drop back to 1200 in a couple of games, so you can't really rig this system. There's another issue right now of people starting from 1000 points,  but the people who made the teams start from 0. This mean 2 to 3 weeks of a difference in farming rating to the detriment of team makers.
5. The gems in PvP for honor are welcome, but extend the purchasable gems with resilience gems. Right now we can only buy PvE gems. The 10 resilience gems from Halaa is nigh impossible to get too, would be nice to buy those as well.
6. This is potentially a stretch, but the dueling community is non-existent. As a huge fans of duels, I would love to see an increase in activity. Make use of the custom arenas you have made in Wetlands and Tanaris Island. Let us queue for duels in exchange for arena points. This could potentially fill the server with PvP during the dead hours when the pop isn't enough to get a BG/arena started.

Regarding balancing: right now PvP feels who can nuke the other side faster. I'm not a fan of custom changes, but it would be nice to get access to WOTLK gems and enchants (for honor) to buff the survivability a bit. We already get access to WOTLK glyphs.

Cheers,
Dumi

**Comment by sassouiabd** [comment]
I hope that this post will swiftly prompt actions instead of other discussions for months. 
Completly aligned with the six propositions, it serves as a solid foundation for uplifting the current dire situation

Ivarx

**Comment by vlad7394** [comment]
I've been playing on the server for a bit over 2 years now. I enjoy PvP'ing and I must admit the whole situation regarding pvp is dire right now. The 10 arena points gain from daily 1st bg win is a very nice addition, but it is truly not enough to sparkle life back into pvp on CC. I agree with all the points Dumika stated above and like Ivarx said, I hope changes will be implemented swiftly rather than letting things stale for several months.
Before ending, I must add that pvp is really unbalanced in this iteration of the game and the feeling of who can burst the other faster wins, is really prevalent and since we already have wotlk talents/glyphs maybe having access to +20 resil/ +30 stamina etc. gems via honor points could help balance current damage vs survivability. (not to mention several specs miss key abilities from their toolkit in order to better face enemies in pvp - this could be an extra points in Dumika's list - I'll write it here)
 - release all the other abilities and spell ranks now, instead of waiting until 80, that way more specs would be viable in pvp (we already have tuning for pve and so I don't think it will be a problem in pve; tuning can always be made stronger if needed). For instance, I only play 1 toon since my time only allows for that and the class I enjoy most is shaman, elemental spec to be specific. Elemental right now lacks both Lava Burst and Hex, 2 key spells in an elemental's pvp toolkit.

This is all I had to say. Hopefully we'll see some changes soon
Vvlad

**Reply by heyitsbench** [reply]
> release all the other abilities and spell ranks now, instead of waiting until 80

I absolutely disagree with this idea. For quite a few people both past and present, the draw of this server has been the whole twink idea, inherent to that is restricted abilities. Even the paladin seal situation during 64 bracket and making both available during that bracket I didn't agree with.

**Reply by balleny** [reply]
same, if you would want to release the higher level abilities, you would need to adjust the damage of all these abilities.
then you can also instead go and adjust all currently available abilities to do reasonable damage in pvp (like reducing revenge damage by 50% in pvp, ambush damage by 60%, etc _(numbers are examples)_

**Reply by vlad7394** [reply]
In my view, the appeal to the server is progressive release of content with minimum bugs, getting the chance to experience older thank wotlk content with wotlk perks. We already have wotlk talents, why not also have wotlk full spectre of abilities? I agree damage abilities would need a % nerf on damage, but other than that, I don't see how it eould hurt the experience, since the experience is definitely not vanilla or tbc.

**Comment by balleny** [comment]
https://github.com/chromiecraft/chromiecraft/discussions/4161
https://github.com/chromiecraft/chromiecraft/discussions/5360
https://github.com/chromiecraft/chromiecraft/issues/6062
https://github.com/chromiecraft/chromiecraft/issues/6038

**Reply by balleny** [reply]
more relevant for me going into a next season would be the addition of s1 arena gear for a combination of arena points + honor.

typically, i would propose a base price of 50k honor and "arena points from 5s + arena points possible to earn from CTA + 1" so that you can get 1 every other week. The inflation with additional 70 arena points per week may have to be accounted for.
gloves should be a bit lower costs, weapons obviously higher

**Reply by dumikachromiecraft** [reply]
I read through those posts. I agree with a lot of things, but WSG and AB requiring 20 people to start would kill PvP here for good. 

As for the pricing of S1 items: your suggestion would take like 10-12 (so like 3 solid months) weeks from ground zero to get full S1 gear. By the time you finish gearing for S1, Season 3 will be knocking on the door. I'd say don't lock previous gear (S1 in this case) behind a weekly reset. Imagine entering arenas in starter blues hoping to progress, and there is Reily in full S2 gear 2 shotting you.

**Reply by balleny** [reply]
i mean the idea is the following (leaving out the honor part for simplicity) - for S1 pieces in S2:
lets say you get 350 AP from 5s and 150 from CTA as a max.
if you need 510 for chest, legs, head and 400 for shoulders, hands; and maybe 800 for weapon (mh) or 1,1k for 2hand, then you would be able to get a setpiece every week if you are active, or a weapon in 2-3 weeks.
if you do the bare minimum
- battlegrounds only: 3 weeks a piece (cause math) -> incentive to play arena, too
- 10 games of 5s: 2 weeks a piece -> still need to do bgs for honor part

if you start playing arena and want to play it a bit more serious, you may reach a rating resulting in 3-4 pieces in 2 weeks if you also add all CTA stuff; allowing new players to catch up quicker or players to play pvp with alts, too - without the stress of reaching a competitive rating to collect some gear

the gear will still not be "free" and reaching full s1 doing pvp on bare minimum / efficiency (10 games 5s and 1 bg win per CTA, skipping 3 days per week), would still require you farm 7-9 weeks for set + all weapon slots.

**Comment by balleny** [comment]
> Kill the queue time between arena games with fire. Seriously. There is no good reason for this to be in with such low arena activity. I only heard complaints about it.

this is in for reasons - but if you want "low" mmr people to get farmed, only so that high mmr teams can get their 10 games, sure remove the mmr search radius (which anyway is a buggy mess cause it has a hardcoded 10 min value at which it takes any team as a opponent independent of mmr)

> The 25 points from CTA and the 10 points from daily wins are appreciated, and while they increase BG activity, they don't do much about arena activity. I propose that you give 10 AP for the first daily arena win per category (2s/3s/5s). This could bring the total to 380 per week from dailies (70 per BG/100 per CTA/210 from arena). That's 724 points per week, so you could get a piece every 3 weeks not every 6 weeks like now.

sounds interesting but instead of dailys, i would suggest making it how it is in cata, weeklies. you get "bonus" for your first x arena games each week

> Release the next tier rings and necklace now, and release the accessories (belt, bracer, boots) with TK. Currently the best PvP rings are from SSC, while actual PvP players are stuck with rings from 5 brackets ago. The next tier jewelry are item level 128, on par with SSC gear.

pvp was always secondclass on chromie and we saw exactly that happen here again. pvp and pve releases should be aligned better

> The main problem with climbing the rankings is population. I have many times reached high rating only to abandon my team, because the partner quit the server from low activity. And the grind starts from 0 again in a new team. Make the points start from 1500. If team is bad, they will drop back to 1200 in a couple of games, so you can't really rig this system. There's another issue right now of people starting from 1000 points, but the people who made the teams start from 0. This mean 2 to 3 weeks of a difference in farming rating to the detriment of team makers.

see tickets above. I would go as far and say have teams start above the 1500 breakpoint. people declining will have an incentive to create a new team (for more arena points) increasing the influx of arena points into the system with each team

> The gems in PvP for honor are welcome, but extend the purchasable gems with resilience gems. Right now we can only buy PvE gems. The 10 resilience gems from Halaa is nigh impossible to get too, would be nice to buy those as well.

or directly making wotlk resilience gems available to counteract imbalances

**Comment by Never6** [comment]
Adding wrath gems and enchants would surely increase PvP activity  but really I don't think that is the best idea for the state of the overall game. Because you have to take into account that it also affects PvE as well. The purpose of the server is supposed to test raids. Adding wrath gems and enchants would really only add onto the issue and create more problems.
Also i think adding wrath resil gems would only buff the already strong classes so burst gets hurt even more
my fully buffed rip crits a fully geared war for 550. If you think adding resil gems is the best way to go then be my guest

**Reply by tbcxd** [reply]
The wrath 20 resil gems would have very little impact on pve.
Stamina gems should 100% NOT be added for honor, only resil ones.

**Reply by Never6** [reply]
ok if its only resil gems then that is fine
someone said make wrath gems and enchants avaible 
when I read wrath gems i assume that means all wrath gems

**Reply by Franaloudzx** [reply]
I'm against any tries of class customization. I want to experience every level as it is, 60/70/80, we will be all at level 80 eventually, so i wouldn't release any wotlk item, glyphs have already been released and that if you ask me, is hard to determine the impact in general so i can't complain.

The meta in every level is different, and that's how it should be imo, different doesn't mean bad. I wanted to enjoy Vanilla and now TBC, and going into it to develop my proper equipment to face PvP and PvE.

Of course i can't disagree that in last phases of TBC things are gonna get more appealing for everyone, since gear will bring stats like Haste and Armor Penetration rating, so classes will again have more variety and the meta might change .

**Reply by balleny** [reply]
fact remains that lvl 70 in wotlk is totally different than level 70 in tbc.
wrath was never balanced around any level below level 80, especially in the patch 3.3.5 we are using. you could argue patch 3.0 would be more viable for our progression but that only holds true for 70 and cannot be applied to 60.

instead, changes to be closer to progression, good and bad changes have been done. but, you even right now, you have either a proper level 70 (and restricted to level 70) type of gameplay (see wrath glyphs as one example) nor have a full wrath like experience (e.g. no higher level gems available to be used in lower level gear)

**Comment by tbcxd** [comment]
1) is something I would really love to see, but with letting people purchase s1 for honor.
A s1 player would 100% stand a good chance vs s2 geared player, but blue rep gear vs s2 is awful.

**Reply by Never6** [reply]
hard agree

**Reply by vlad7394** [reply]
I agree with S1 gear being available for honor points only. I have been active through arena S1 and have done arenas on a weekly bases, but I don't mind people being able to get their gear with honor points/tier tokens once S2 will release.
Pvp is imbalanced as is, if gear gap will also be a problem, late comers to arena/pvp will simply not do arenas.

**Comment by Franaloudzx** [comment]
I'm gonna say this clear, the peak of PvP in every sense is doing arenas, and if we go deeper into it, the 3vs3 bracket is the main /most relevant one. That's why i'm gonna focus on what from my pow is far from being "active" compared to battlegrounds. That being said, here are my main ideas that might work objectively in any private server:

Ideas to boost 3vs3 bracket (might also work for 2vs2 if properly applied):

1. Easy ones:
- Giving extra points to the 3v3 bracket by creating a quest that requires 10 wins per week (the amount is irrelevant, but ofc should be appealing enough, like 100 points).

- Rating Requirements: to boost both 2vs2 and 3vs3 The gear should require arena rating, not only arena points. Since many ppl love wotlk in every sense, i remind you that in wotlk gear requires arena rating... but ppl tend to "select" the most convenient stuff, or that's what the playerbase showed me lately...

- 3vs3 being a must to get best pvp gear: let me elaborate this idea. Every phase we have 2 PvP sets, one semi-bis and the other is bis. So i would put the semi-bis one only requiring 0 rating or 2vs2 rating, and the bis one would require 3vs3 rating.

This would be an example:

Phase 2:
  Arena season 1 needs 0 to some 2vs2 rating.
  Arena season 2 needs 3vs3 **rating and arena points** (like in wotlk, gloves 1400, chest 1500, pants 1600, weapons 1800, helmet 1950, shoulders 2k, tabard 2.2k)

Phase 3:
   Arena season 1: Honor points.
   Arena season 2 needs 0 to some 2vs2 rating.
  Arena season 3 needs 3vs3 **rating and arena points** (like in wotlk, gloves 1400, chest 1500, pants 1600, weapons 1800, helmet 1950, shoulders 2k, tabard 2.2k)

Last PvP phase:
  Arena season 2: Honor points
  Arena season 3 needs 0 to some 2vs2 rating and arena points.
  Arena season 4 needs 3vs3 **rating and arena points** (like in wotlk, gloves 1400, chest 1500, pants 1600, weapons 1800, helmet 1950, shoulders 2k, tabard 2.2k)

This should be enphasized in s4 since is going to overpower tier 6 and i believe the server also wants to encourage ppl to do PvE content.

The side effects of this idea is what most of us know, ppl /Cry 24x10^7.

- Hard to script and well known:

Soloqueue 3vs3: i already know the answer, i just put it here if any miracle happens.

P.D.: I would play (and also many friends) way more bgs if group queue was enabled. Just as a polite reminder.

P.D. 2: I know the community isn't ready for this idea since most ppl is far from being a tryhard in PvP, i just wanna say i only played in weekends and reached 1.8k and started from blue to epic gear step by step.

 I didn't get for free my gear, and when i see someone with semi-bis or bis gear, what i wish to say into my mind is "wow look at this guy, must be very good in PvP: respect!", what i don't wish to say is: "look another casual player with bis PvP gear, just like any other player...".

I want to bring the feeling of achieving something and excitement. Arenas never felt like any grinding stuff, for me it feels like every arena game can be completely new, is a refreshing experience that never gets old. That's why i PvP over PvE, but also that's why i do arenas, bcs is a mix of cooldown strategy and innovation.

After all that being said, thankyou very much for reading me.

And see you ingame. 

Much love.

Franagan.

**Reply by tbcxd** [reply]
I would love to see s2 requiring rating to obtain!
Be it from 2v2 or 3v3, but no way I would agree if one could just get carried in 5s by the best players here and have an easy time gaining fully geared.
Rating requirements would make people work harder to get their gear, but might at the same time be counterproductive if the cooldown between games doesn't get removed.
I don't think there's anyone who likes it anyway, it could be nice if the pvp community was in the hundreads, but it's not.

People might argue it's unfair against people who are just starting to get geared, but that can be fixed if these players can buy s1 gear for ~300ish arena points with a 30% increasing of weekly arena points a player can obtain.
In less than a month, that should get the player fully s1 geared, the said player should stand a good chance even if his enemies are s2 geared.

It's awful to grow your rating once you hit 1.5k+ rating, combine the small gain from fighting lower ranked teams with the arena cooldown and you got yourself some of the most time consuming grinds.
God forbids you lose 1 game, then you're set back days worth of games.

**Reply by sassouiabd** [reply]
How would you get 2k2 with +1 and - 20. Without reviewing the mmr system, the rating requirements will not works.
Regarding soloq, i have seen it not working on servers with 5k population, i doubt it will work on chromiecraft. In order to implement it you need to put constraints on the players selections and if currently we are not able to solve the bg queuing system we can forget about going into this path.

Weekly quests for arena points is good idea, i have been saying this since the start.

The first changes, i would prefer seeing personally is : 
- no more arena water at 90 silver each
- no more 5min queues between games
- reviewing arena gear costs
- removing damage and healing réduction and replacing it with +20% health (both in bg and arena) blizzard is doing that in SOD currently for the same reasons.
- removing the command lla queue to see the teams
- reducing the amount of lost when higher team per up with lower mmr teams
- fixing the stealth eyes in arena at 90 sec and not 60 sec
- puting previous arena gear with small arena pts and honor

All the changes below doesn't require to be a genius to do them and are no brainer for me and good foundation to move forward.

**Reply by vlad7394** [reply]
As I mentioned in a comment down below, I disagree with arena rating requirement for arena gear. You will find the reason in the comment.
In my opinion, in every suggestion that we bring up regarding arena, we must take into consideration the current state of imbalance which is due to our server's iteration of the game. We have TBC gear with WOTLK talents and a select few specs/classes have the key pvp abilities and imbalanced damage vs surv. ratio to make them op, whilst others struggle very much to even be relevant, let alone climb consistently.

I agree with insentives for reaching a number of wins weekly or some other sort of reward, but I believe gatekeeping gear behind a rating requirement in an unfair state of affairs will hurt arena scenario, not help it. Few suggestions that have already been stated:

 - no more 5 min queues - right now, with the small numbers of active queues, if the rating difference between 2 teams is as little as 110+ rating, it will take 50 minutes to complete the weekly minimum 10 matches. Why have the cooldown system when there are so few arena teams that consistently play?
 - incentive for a number of matches won/weekly OR ratio between wins/losses - this coped with keeping arena gear on sale ONLY for arena points will help keep arena seasoned players pleased because they will be able to get their gear faster and will also help the arena casuals still get their gear, but much slower, ofc. Point is, everybody gets their gear, sooner or later, rather than just casuals never getting their gear in the current season. It took me almost an entire season to get the arena gear with the points you get for the minimum weekly matches and I'm not complaining. I accept the current state and I'm playing my best, but do not gatekeep me from getting the gear only because I chose to play an unpopular spec for the current state;
 - make S1 arena gear available for honor points only OR discounted arena points and honor points - this is probably better, it will make people play a bit of arena at least at the start of the new season;
 - erase all arena points at the start of each new season;
 - make new arena teams start at 1500;
 - what Dumika said about dueling perks - i think that sounds really good.

**Comment by vlad7394** [comment]
Up until we will have WOTLK released, I will disagree with arena rating being required to buy arena gear and here is why:
- the current state of pvp is dominated by imbalance, making a few classes/specs op in comparison to the others (see warriors, enha shamans), thus leading to all pvp match-ups being dominated by these specs;
- this being said, the other classes/specs that want to have a shot at arena gear will just be denied due to inability to climb arena rating;
- I am all for creating an environment in which I would have to be skillful and work to get the gear that I want, but right now it's not that much about skill, as it is about dmg vs survivability not being balanced, several specs missing key abilities still etc.;
- if arena rating requirement will be implemented in order to buy arena gear, for the reasons above, all arena scenarios will be dominated by the few specs I mentioned and players that enjoy other specs will just drop arena (which will probably be my case also) because they will be denied the fair chance of getting arena gear; people will even stop playing for the minimum requirement of 10 games/week to get their weekly points because they won't mean anything w/o rating.

I am on board with incentives like bonus points for a certain number of wins or so, but I think arena rating will not be lucrative (in the current state of content/server) for making pvp popular.
This is my point of view.
Vvlad

**Reply by tbcxd** [reply]
You must be delusional if you think only enh and warriors can get high rating.
Mages/rogues/paladins/druids/priests/hunters can all get rating ez mode.
Only spec I can think of that should struggle is ele shaman, which will most likely get their fair chance once we get past s2.
All other specs look very viable to me

**Reply by vlad7394** [reply]
Why do I have to be the one delusional when you did not understand what I wrote?
I said enhas and warriors are op. I did not mention the other specs because they're not op and also they do not struggle, like you said, they are perfectly viable for climbing.
The one spec that is struggling is indeed elemental shaman and it's the spec I play right now.
The issues I adressed though, being high dmg vs surv., missing abilities etc. apply all across the board, imo.

**Reply by Hurgon** [reply]
And yet if you look at higher rated arena teams you will only see arms warriors, ret and holy palas, couple enha, ambush rogues and disci priest. Odd mage here and there and exactly one druid. No elementals, no hunters, no ferals etc. Also vast majority of high rated players are either paladins or warriors, you may look at all specs as viable to your will, but facts are that theres gross imbalance that favours some classes, and you should know that well considering your experience at 70 twinking. Gatekeeping s2 gear by rating is horrible idea at this server - you will see like 6 people and their alts sporting 1800+ rated items due to gross imbalance and only meta comps being viable at climbing when single loss costs dozen of wins, which translates to week of progress.  Last thing we need here imo is lowering pool of players atracted to arena by gating gear by required rating - my guess would be that people playing off meta classes would just play those 10 games to get points, supplement that by battleground and be done with their 2/3 low rated pieces, never going to arena again.

**Reply by Franaloudzx** [reply]
Yeah, i mean come on, let's be honest, it's not bcs 80% is mostly PvE dedicated playerbase and as a direct consequence PvP players are mainly bgs "mainstream". Sure, but then why playing bgs either if you don't like the balance of the bracket?? I'm not even saying you have to play 2vs2, but pointing the need of giving 3vs3 more attractiveness in reward terms, to get bis gear (that is by far the most balanced bracket, and i bet 99% of the PvP community haven't even joined a single game). Literally playing bgs means u are not even looking for a balanced fight, you are just looking for casual pvp. Even if it's relevant or not the balance for the generic player, giving the gear for honor points or rewarding arena points by doing battlegrounds, is not gonna boost the arena activity, and we had an entire season to see (i believe 8 months) that people were only worried about their 10 weekly arena games and then going back to yolo PvP (battlegrounds). What in theory means, that they need a "reason" to stick into queue of arenas. And what best reason than asking for arena rating and in the 3vs3 bracket where most ppl might enjoy a real balance.

Don't lie to yourself, people is not queueing more arenas because they don't **need** to, because they have nothing to lose and eventually they will get bis, but if you create a **need**, they will response to that need either rejecting it or they will answer generating a hidden population that instead of being in bg 24/7, will start queueing arenas trying to reach what is needed.

Then by generating more teams in queue, everyone gets benefits from it since it will allow some people to reach more rating than they did before (the more teams, more chances to win and different mmrs are involved, that's how in high populated servers ppl even reach 3k arena rating).

This is my answer, and believe it or not, the balance is all subectivity, but i think its logic to believe that 2v6 or 4v5 is less balanced than a 3vs3 organised and coordinated team.

Long life to meritocracy! 🤟

Thankyou for your feedback.

Franagan.

P.D: The 5 minutes queue only happens to avoid wintrading abuse and when only 2 arena teams are queued.

P.D 2: i have been playing with a frost mage as a partner, he didn't have haste, didn't have enhanced counterspell, didn't have many stuff and still, STILL enjoy the bracket and his class, everyone lacks from skills but we don't give up so easily, we are all adapting.

P.D. 3: I wonder at lvl 80, what top teams gonna be, but i bet is not gonna be much different from what is now.

Once again, thankyou all for reading.

See you ingame!

**Reply by tbcxd** [reply]
Here's the thing with the rating requiremeants frana is proposing.
We don't have to use any blizzlike values to increase higher rating arena activity, something like 1650 rating should be obtainable by ALL specs and comps, it's nothing crazy hard, if everyone's goal is to get to that rating or past it, there will be a lot more teams in that range.

And for casuals who can't play well enough to get to that rating, they can just get their s1 gear and still do very good in any pvp scenario.

I do think a decent rating requirement in order to buy s2 gear would be a good idea and improve the arena activity. 

Right now and for the past 3ish month's that I've been playing here for, I only see "lets cap 10 games and come back next week" there are only 2 or 3 teams who actually try to increase their rating, which is really sad.

And those teams get slapped in the face by 5 mins cooldown, which makes everything worse.
This cd could be cut in half, imo.

Everyone talks about people having no reason to queue more than 10 games a week, there's your reason, trying to get higher rating in order to buy THE BEST pvp gear available.

I would also love to see 5v5 getting removed or have its arena points gain reduced.
I have stated multiple times that 5v5 isn't really "arena" is just a faceroll activity that barely requires any skill.
You can stack 2 or 3 players of the same class and spec, just pop starfall and watch stuff die in unhealable dmg, if that's skill, then I must change how I view wow pvp and I've been living through lies for all these years.

**Reply by Hurgon** [reply]
Hm, i admit i could be wrong before with that statement of opposing gear rating requirements, but imo they should be low enough to be attainable by dedicated players of all classes, like 1600/1700 for shoulders, head and 1800 for weapons, and of course that would be 3s not 2s requirement. 5s can burn for all i care, its meme bracket anyway and it takes players away from 2s 3s.  Also big thing would be doing  that daily arena quest for 20 ap for like 5 games played - not necessarily wins, maybe even instead of daily bg quest if we dont want ap inflation to hit too soon.

**Reply by tbcxd** [reply]
That sounds really good.
5 games for 20 AP sounds really spicy:D

**Reply by vlad7394** [reply]
P.D.1: Hey, me playing as ele shaman I believe it's a statement to the fact that I do not give up easy xD.
P.D.2: A mage is still inherently better equipped for pvp than an ele at this stage of the game.
Anyways, now that you put it this way, like Hurgon said, I might even think it's a good idea. Having a rating requirement that's just low enough for every class/spec to have a shot at it, sounds appealing. And yes, I like 3s bracket, it's better balanced than 2s and besides climbing rating, if there's another incentive to play arena, the better. Like, only take a look at what daily 10ap gain for 1st bg win of the day did to bg queues. Give it, it's only the 1st week, but I think it's bgs will pop more tahn they used to.

**Comment by TotallyNotMyAccount** [comment]
1. I think a small waiting time is needed, just to prevent the same teams always playing each other. But if you only play your 10 games to cap that's 50 minutes of pure waiting time right there. It's terrible! Cut it down to 30-60 sec per game please.

2. While I like the idea of incentivizing more arena participation, I worry that giving out too many arena points is going to eventually kill participation of the more casual, BG oriented pvp’ers. We certainly didn’t get enough points at the start of S1, but with CTA and daylie BG that’s an additional 170 pts / week that anyone can pick up. Frankly put I’d prefer if the 10 daylie win points were shifted from BGs to 3v3 arena. 2v2 needs far less help and 5v5 has enough incentives to play it already.

3. Yeah, the next tier of honor gear should be released with S2. I expect this to happen anyway, but if it doesn’t, that’s a misstep by the devs for sure.

4. Fresh teams being at 1k sounds far more reasonable. You should at least have to climb a little before you get good points.

5. Adding the Halaa resil gem and maybe dungeon epic gems that have resil and/or spell pen on them to the honor vendor sounds like a good change.

6. Even more arena points gained from something other than arena. Not a fan at all. Also fucks over healers.

7. The only wrath thing I could see them making available without major balance and PvE repercussions are the pure resil gems. Even that would be a stupid bandaid fix, and also cut into jewelcrafter’s margins. Ivar mentioned a 20% health increase across the board as a solution. As long as it’s only applied in instanced PvP, that sounds much more reasonable.

As a final note: Minimum ratings on arena items have been mentioned. I also think forcing people to climb a bit before they get access to the very best pvp gear is warranted and should drive up participation as long as the rating needed is reasonable. I think weapons are too important to gate them behind high ratings, and shoulders + helmet work way better as low impact (stat wise) but highly visible status items available at higher ratings. 
Something like: 
```
Item		-	Min Rating
Weapon		-	1550
Helmet		-	1700
Shoulders	-	1850
```
